889 TC- WalkerColina -02

wcolina opened the pull request:
Test case #41 in the excel file.

**Test case:**
![image](https://user-images.githubusercontent.com/852076/41891105-d4d92110-78e0-11e8-86cc-75449ddff30f.png)

**Automated test case result:**
![image](https://user-images.githubusercontent.com/852076/41891038-6c744db6-78e0-11e8-8256-437252786b34.png)

wcolina committed:
Added automated test case for:
TC-44 Clasificacion de equipo de proteccion personal - Todos los datos de clasificacion de equipos de proteccion personal son mostrados en una lista.

wcolina committed:
Added automated test case for:
                                                                                                                                      TC-44 Clasificacion de equipo de proteccion personal - Todos los datos de clasificacion de equipos de proteccion personal son mostrados en una lista.

Refactor code with more appropriate variable and method names.

wcolina committed:
Format code.
Improved imports.
Using abbreviation PPE for personal protection equipment in the package.
Using PPE acronym in method's name.
Refactor method names following correct format, deleted word 'On' and changed 'Click' instead of 'Clicking' in the steps/feature definitions.

wcolina committed:
Fixed feedback in the review.

wcolina committed:
Fixed feedback in the review.

wcolina committed:
Addes cosntructor in page.

wcolina committed:
Merge remote-tracking branch 'remotes/origin/development' into TC-44.1

# Conflicts:
#	src/main/java/org/umssdiplo/automationv01/core/managepage/home/SSIAHome.java
#	src/test/java/org/umssdiplo/automationv01/stepdefinitionproject/ssia/SSIAStepDefinitions.java

wcolina committed:
Merge remote-tracking branch 'remotes/origin/development' into TC-44.1

# Conflicts:
#	src/main/java/org/umssdiplo/automationv01/core/managepage/home/SSIAHome.java
#	src/test/java/org/umssdiplo/automationv01/stepdefinitionproject/ssia/SSIAStepDefinitions.java

wcolina committed:
Automated test case for La inserción de registro Clasificación de equipo de protección personal se muestra en la lista de Clasificación de equipo de protección personal cuando presiono botón Save.

wcolina committed:
deleted unnecesary files.

wcolina committed:
deleted unnecesary files.

airmiky commented on src/main/java/org/umssdiplo/automationv01/core/managepage/ppe/PPEClassificationCreate.java:
```diff
@@ -0,0 +1,47 @@
+    private WebElement saveButton;
+
+    public PPEClassificationCreate() {
+        CommonEvents.isPresent(saveButton);
```
usar CommonEvents.isVisible(saveButton);

wcolina replied:
Done.

airmiky commented on src/main/java/org/umssdiplo/automationv01/core/managepage/ppe/PPEClassificationList.java:
```diff
@@ -0,0 +1,38 @@
+    private WebElement lastPPEClassificationName;
+
+    public PPEClassificationList() {
+        CommonEvents.isPresent(ppeClassificationContainer);
```
Use CommonEvents.isVisible(ppeClassificationContainer);

wcolina replied:
Done.

airmiky commented on src/main/java/org/umssdiplo/automationv01/core/managepage/ppe/PPEClassificationList.java:
```diff
@@ -0,0 +1,38 @@
+    }
+
+    public String getLastPPEClassificationNameInTable() {
+        return lastPPEClassificationName.getText();
```
Utilizar CommonEvents.getTextContent(lastPPEClassificationName);

wcolina replied:
Done.

airmiky commented on src/test/java/org/umssdiplo/automationv01/stepdefinitionproject/ssia/SSIAStepDefinitions.java:
```diff
@@ -105,13 +113,51 @@ public void clickAccidentMenu() throws Throwable {
         Assert.assertTrue(accidentList.isAccidentListPresent(), "Fail, Accident List is not loaded");
     }
-    // PPE List
+
```
Eliminar espacio

wcolina replied:
Done.

airmiky commented on src/test/java/org/umssdiplo/automationv01/stepdefinitionproject/ssia/SSIAStepDefinitions.java:
```diff
@@ -105,13 +113,51 @@ public void clickAccidentMenu() throws Throwable {
+    }
+
+    // Existing PPE List
+
```
Eliminar espacio

wcolina replied:
Deleted all comment.

airmiky commented on src/test/resources/features/ppe/PPEClassificationCreate.feature:
```diff
@@ -0,0 +1,12 @@
+
+  Scenario: La inserción de registro Clasificación de equipo de protección personal se muestra en la lista de Clasificación de equipo de protección personal cuando presiono botón Save
+    Given 'SSI-A' home page is loaded
+    And Click 'PPE' menu on 'Header' page
```
Al inicio utilizar minuscula "click"

wcolina replied:
Done.

airmiky commented on src/test/resources/features/ppe/PPEClassificationCreate.feature:
```diff
@@ -0,0 +1,12 @@
+  Scenario: La inserción de registro Clasificación de equipo de protección personal se muestra en la lista de Clasificación de equipo de protección personal cuando presiono botón Save
+    Given 'SSI-A' home page is loaded
+    And Click 'PPE' menu on 'Header' page
+    And Click 'PPE Classification' sub menu of 'PPE' menu
```
Al inicio utilizar minuscula "click"

wcolina replied:
Done.

airmiky commented on src/test/resources/features/ppe/PPEClassificationCreate.feature:
```diff
@@ -0,0 +1,12 @@
+    Given 'SSI-A' home page is loaded
+    And Click 'PPE' menu on 'Header' page
+    And Click 'PPE Classification' sub menu of 'PPE' menu
+    And Click 'New PPE Classification' button of 'PPE Classification list' page
```
Al inicio utilizar minuscula "click"

wcolina replied:
Done.

airmiky commented on src/test/resources/features/ppe/PPEClassificationCreate.feature:
```diff
@@ -0,0 +1,12 @@
+    And Click 'PPE' menu on 'Header' page
+    And Click 'PPE Classification' sub menu of 'PPE' menu
+    And Click 'New PPE Classification' button of 'PPE Classification list' page
+    And Fill 'PPE Classification' form using Data Driver Test on create 'PPE Classification' page
```
Al inicio utilizar minuscula "fill"

wcolina replied:
Done.

airmiky commented on src/test/resources/features/ppe/PPEClassificationCreate.feature:
```diff
@@ -0,0 +1,12 @@
+    And Click 'PPE Classification' sub menu of 'PPE' menu
+    And Click 'New PPE Classification' button of 'PPE Classification list' page
+    And Fill 'PPE Classification' form using Data Driver Test on create 'PPE Classification' page
+    And Click 'Save' button into create 'PPE Classification' form page
```
Al inicio utilizar minuscula "click"

wcolina replied:
Done.

airmiky commented on src/test/resources/features/ppe/PPEClassificationCreate.feature:
```diff
@@ -0,0 +1,12 @@
+    And Click 'New PPE Classification' button of 'PPE Classification list' page
+    And Fill 'PPE Classification' form using Data Driver Test on create 'PPE Classification' page
+    And Click 'Save' button into create 'PPE Classification' form page
+    Then Created 'PPE Classification' is showed in PPE Classification list page
```
Al inicio utilizar minuscula "created"
comillas simples en 'PPE Classification list'

wcolina replied:
Done.

airmiky commented on src/test/resources/features/ppe/PPEClassificationList.feature:
```diff
@@ -0,0 +1,9 @@
+
+  Scenario: Clasificacion de equipo de proteccion personal - Todos los datos de clasificacion de equipos de proteccion personal son mostrados en una lista
+    Given 'SSI-A' home page is loaded
+    And Click 'PPE' menu on 'Header' page
```
Al inicio utilizar minuscula "click"

wcolina replied:
Done.

airmiky commented on src/test/resources/features/ppe/PPEClassificationList.feature:
```diff
@@ -0,0 +1,9 @@
+  Scenario: Clasificacion de equipo de proteccion personal - Todos los datos de clasificacion de equipos de proteccion personal son mostrados en una lista
+    Given 'SSI-A' home page is loaded
+    And Click 'PPE' menu on 'Header' page
+    And Click 'PPE Classification' sub menu of 'PPE' menu
```
Al inicio utilizar minuscula "click"

wcolina replied:
Done.

airmiky commented on src/test/resources/features/ppe/PPEClassificationCreate.feature:
```diff
@@ -0,0 +1,12 @@
+# @Author: Walker Colina
+# feature to review behaviour of create ppe classification
+Feature: PPEClassificationCreate
+
```
Eliminar espacio

wcolina replied:
No changes here. The blank line is generated for the ''Reformat Code' option.

airmiky commented on src/test/resources/features/ppe/PPEClassificationList.feature:
```diff
@@ -0,0 +1,9 @@
+# @Author: Walker Colina
+# feature to review behaviour of ppe classification list
+Feature: PPEClassificationList
+
```
Eliminar espacio

wcolina replied:
No changes here. The blank line is generated for the ''Reformat Code' option.

wcolina replied:
Restored chromedriver and geckodriver files deleted by error

airmiky reviewed (commented):
Revisar las observaciones por favor

wcolina committed:
Automated test case for:

1. Clasificación de EPP - Lista - Todos los datos de clasificación de equipo de protección personal son mostrados en una lista después de seleccionar la opción "PpeClassification" del menú "PPE"
2. Clasificación de EPP - Registro - Después de registrar datos de una clasificación de equipo de protección personal, estos se despliegan en la lista de clasificación de equipo de protección personal

wcolina commented:
Fixed all mentioned in the feedback.

wcolina commented:
Renamed test case title.

wcolina committed:
Automated test case for:

1. Clasificación de EPP - Lista - Todos los datos de clasificación de equipo de protección personal son mostrados en una lista después de seleccionar la opción "PpeClassification" del menú "PPE"
2. Clasificación de EPP - Registro - Después de registrar datos de una clasificación de equipo de protección personal, estos se despliegan en la lista de clasificación de equipo de protección personal

wcolina committed:
Restored files deleted by error.

wcolina commented:
Restored chromedriver and geckodriver files deleted by error
